.github/workflows/link-checker.yml

```diff
@@ -0,0 +1,47 @@
+name: link checker
+
+on:
+  workflow_dispatch:
+  # Check links once a week on Monday
+  schedule:
+    - cron: "00 00 * * 1"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write # required for peter-evans/create-issue-from-file
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          args: >
+            --verbose
+            --no-progress
+            --include-fragments
+            --accept 100..=103,200..=299,429
+            --exclude 408,504
+            --exclude-loopback
+            --exclude-path 'presto-docs/src/main/sphinx/_templates/partials/toc.html'
+            --exclude 'https://www.gnu.org/software/gperf'
+            --max-concurrency 32 
```

Comment:
Not sure about the implementation, but our GH actions runners only have 4 cores. Of course some request can be in-flight while others are waiting on IO/being processed but just wanted to check if you experimentd with this value